src/components/Pagination.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
```

Comment:
remove this line of code we don't need it in React 18.2

Reply:
Done.